### ISSUE TITLE
Xcode 7 NS_ENUM comments using @brief

### ISSUE THREAD

**Issue by nspavlo** [onevcat/VVDocumenter-Xcode#184]
Hello!
If using ```@brief``` is enabled, typing ```///``` before enum crashes Xcode.
```typedef NS_ENUM(NSUInteger, TestProto) {
    TestProto1 = 0,
    TestProto2
};```

Using Xcode Version 7.0.1 (7A1001)
Alcatraz 1.1.8

**Comment by onevcat** [comment]
Thanks for the report, I will look at it soon.

**Comment by calios** [comment]
I have the same issue, too. Xcode 7.0.1. And, thanks for checking it.

**Comment by onevcat** [comment]
Should be fixed in #186
